lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -672,6 +672,8 @@ defmodule Kernel.SpecialForms do
   defmacro __CALLER__
 
   @doc """
+  Pin operator.
```

Comment:
I prefer the previous summary as first line. We can follow it by: "Also known as the pin operator".